lib/bridge/IrcBridge.js

```diff
@@ -518,6 +527,32 @@ IrcBridge.prototype.sendMatrixAction = function(room, from, action, req) {
     return Promise.reject(new Error("Unknown action: " + action.type));
 };
 
+IrcBridge.prototype._incrementMessageInFlight = function(room, virtualUser) {
+    const key = `${room.getId()}:${virtualUser.getId()}`;
+    if (this._messagesInFlight[key] === undefined) {
```

Comment:
minor thing: might as well make this a `!this_messagesInFlight[key]` rather than explicit type checking.

Reply:
I'd rather make it more explicit than `!` ?